quiz/quiz-questions.js

```diff
@@ -1,269 +1,263 @@
 const questionTemplate = {
-  question: "",
-  answers: ["", "", "", ""],
-  correctAnswer: "",
+  question: '',
```

Comment:
Unfortunately, all of the double quotes were changed to single quotes.  This cannot be merged.